nix/sources.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This is a generated file. You don't need to edit it manually.

Bump you nixpkgs definition with:

```
nix-shell -p niv --run 'niv update nixpkgs -b nixpkgs-unstable'
```
from the root of the project. It will automatically update this file for you.

Reply:
@garyverhaegen-da so that is the command to regenerate this file when version upgrade is needed?